### ISSUE TITLE
detect_chromedriver_version=true getting incorrect version if dev/canary/beta installed

### ISSUE THREAD

**Issue by kylemorelock** [giggio/node-chromedriver#364]
Running this awesome util (thanks for publishing) and due to some recent changes, the detect_chromedriver_version=true option is causing a failure when downloading chromedriver. The util is detecting version 104.0.5071.0 and since there is no published chromedriver version, the download fails with a 404. 

Where is the util reading the chrome version from? Perhaps an improvement would be to read from a different, channel specific, location?

**Comment by giggio** [comment]
Yes, it'll probably fail. It is coming from https://github.com/testimio/chrome-version/.
I checked, they don't have an option to select the channel at the moment. You could open an issue there, and once it is implemented we could update our code to take it into consideration.
Or maybe search for another library, I'm open to that as well. PRs are welcome.

**Comment by kylemorelock** [comment]
Thanks. Issue opened there. https://github.com/testimio/chrome-version/issues/13
